api/v1alpha1/lvmcluster_types.go

```diff
@@ -65,10 +65,15 @@ type DeviceSelector struct {
 // type LVMConfig struct {
 // 	thinProvision bool `json:"thinProvision,omitempty"`
 // }
+
 // LVMClusterStatus defines the observed state of LVMCluster
 type LVMClusterStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
+
+	// ready describes if the LvmCluster is ready.
+	// +optional
+	Ready bool `json:"ready,omitempty"`
```

Comment:
This is just a placeholder field in the status. We will probably end up removing this.